src/import/sst_service.rs

```diff
@@ -155,6 +160,57 @@ where
         None
     }
 
+    fn build_apply_request<'a, 'b>(
+        &self,
+        reqs: &'a mut Vec<Request>,
+        cmd_reqs: &'a mut Vec<RaftCmdRequest>,
+        is_delete: bool,
+        cf: &'b str,
+        context: Context,
```

Comment:
`Context` may be a reference, or perhaps we can remove the `clone` in L190 and L207?

Reply:
if `Context` become a reference, then `make_request_header` need another clone for Context. 🤔

Reply:
So maybe just remove those clones? (will rustc complain with that?)

Reply:
since closure is FnMut，so we can not just remove those clones.🤔